tests/test_to_hepmc3.py

```diff
@@ -63,8 +51,8 @@ def test_to_hepmc3(Model):
         # in case of overlapping ranges of incoming particles
         # the earlier vertex keeps them
         for a, b in unique_vertices:
-            if pa != (a, b) and a <= pa[0] < b:
-                pa = b, pa[1]
+            if pa != (a, b) and a <= pa[0] <= b:
+                pa = b + 1, pa[1]
```

Comment:
Where does this change come from?

Reply:
This fixes the problem with one of the asserts that compared hempc result with the result of the code that imitates (as I understand) one of the hepmc algorithms. The problem was that before, for example, [1, 5] region with [1, 2] existing region was devided to [1, 2] and [2, 5]. It gave an overlapping ranges. The fix devides [1, 5] to [1,2] and [3, 5] ranges.